composer/content.md

```diff
@@ -91,42 +92,42 @@ Our image is aimed at quickly running Composer without the need for having a PHP
 
 Suggestions:
 
--	use [`--ignore-platform-reqs`](https://getcomposer.org/doc/03-cli.md#install-i):
-
-	```console
-	$ composer install --ignore-platform-reqs
-	```
-
--	specify the target [platform](https://getcomposer.org/doc/06-config.md#platform) in your `composer.json`:
+-	(recommended) specify the target [platform](https://getcomposer.org/doc/06-config.md#platform) in your `composer.json`:
 
 	```json
 	{
 	  "config": {
 	    "platform": {
-	      "php": "7.1.3"
+	      "php": "MAJOR.MINOR.PATCH"
 	    }
 	  }
 	}
 	```
 
+-	use [`--ignore-platform-reqs`](https://getcomposer.org/doc/03-cli.md#install-i):
```

Comment:
Can we add a warning? Something like "This option is potentially harmful. You might get dependencies that are incompatible with your target environment."

Reply:
Labeled the options in a suggestive manner. Good enough?

Reply:
I'd still prefer a warning, but your solution works for me. Thank you!